library/a-star/metadata/8th-floor-hall

```diff
@@ -0,0 +1 @@
ull,"H-820","H-820","H-820","H-820","H-820",null,null,"H-823","H-823","H-823","H-823"],["H-859","H-859","H-859","H-859",null,"H-860-01",null,"H-860-04","H-862","H-862","H-840","escalator","escalator",null,"H-820","H-820","H-820","H-820","H-820","H-820",null,"H-823","H-823","H-823","H-823"],["H-857","H-857","H-857","H-857",null,"H-860-03",null,"H-860-04","H-862","H-862","H-840","escalator","escalator",null,"H-820","H-820","H-820","H-820","H-820","H-820",null,"H-825","H-825","H-825","H-825"],["H-857","H-857","H-857","H-857",null,"H-860-03",null,"H-860-06","H-862","H-862","H-840","escalator","escalator",null,"H-820","H-820","H-820","H-820","H-820",null,null,"H-825","H-825","H-825","H-825"],["H-857","H-857","H-857","H-857",null,"H-860-05","H-860-03","H-860-06","H-862","H-862","H-840","escalator","escalator",null,"H-832-06","H-832-06","H-832-06","H-822","H-822","H-822",null,"H-827","H-827","H-827","H-827"],["H-855","H-855","H-855","H-855",null,"H-854","H-854","H-854","H-842","H-842","H-840","escalator","escalator",null,"H-832-06","H-832-06","H-832-06","H-822","H-822","H-822",null,"H-827","H-827","H-827","H-827"],["H-855","H-855","H-855","H-855",null,"H-854","H-854","H-871","H-842","H-842","H-840","escalator","escalator",null,"H-832-06","H-832-06","H-832-06","H-822","H-822","H-822",null,"H-827","H-827","H-827","H-827"],["H-853","H-853","H-853","H-853",null,"H-852","H-852","H-871","H-871","H-881","H-840","H-838-01","H-838-01",null,"H-832-03","H-832-06","H-832-02","stairs","N/A","N/A",null,"H-829","H-829","H-829","H-829"],["H-853","H-853","H-853","H-853",null,"N/A","N/A","stairs","H-881","H-881","H-838","H-838","H-838",null,"H-832-03",null,"H-832-02","stairs","N/A","N/A",null,"H-829","H-829","H-829","H-829"],["H-851-02","H-851-02","H-851-01","H-851-01",null,"N/A","N/A","stairs","H-881","H-881","H-838","H-838","H-838",null,"H-832-01",null,"H-832-02","stairs","N/A","N/A",null,"H-829","H-829","H-829","H-829"],["H-851-03","H-851-03",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"H-829","H-829","H-829"],["H-849","H-849","H-849","H-849","H-847","H-847","H-845","H-845","H-843","H-843","H-841","H-841",null,null,"H-837","H-837","H-837","H-835","H-835","H-833","H-833","H-831","H-831","H-831","H-831"],["H-849","H-849","H-849","H-849","H-847","H-847","H-845","H-845","H-843","H-843","H-841","H-841","H-841",null,"H-837","H-837","H-837","H-835","H-835","H-833","H-833","H-831","H-831","H-831","H-831"],["H-849","H-849","H-849","H-849","H-847","H-847","H-845","H-845","H-843","H-843","H-841","H-841","H-841",null,"H-837","H-837","H-837","H-835","H-835","H-833","H-833","H-831","H-831","H-831","H-831"],["H-849","H-849","H-849","H-849","H-847","H-847","H-845","H-845","H-843","H-843","H-841","H-841","H-841",null,"H-837","H-837","H-837","H-835","H-835","H-833","H-833","H-831","H-831","H-831","H-831"],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null]]
```

Comment:
should use 8th floor test instead but switch it to this name to match naming conventions